docs/sources/explore/trace-integration.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Tracing in Explore"
+description = "Tracing in Explore"
+keywords = ["explore", "trace",]
+aliases = ["/docs/grafana/latest/features/explore/"]
+weight = 20
++++
+
+# Tracing in Explore
+
+Explore allows you to visualize traces from tracing data sources. This is available in Grafana v7.0+.
+
+Supported data sources are:
+
+- [Jaeger]({{< relref "../datasources/jaeger.md" >}})
```

Comment:
Please put this list and the one at the top of Logs in Explore in alphabetical order.

Reply:
Done

Reply:
Done.

Reply:
Done.